docker/fluent/Dockerfile

```diff
@@ -1,23 +1,41 @@
-ARG baseImage=centos:7
-FROM ${baseImage} as fluentbuilder
-COPY fluent.tgz /ansys_inc/
-RUN tar -xzf /ansys_inc/fluent.tgz --directory /ansys_inc/ && rm -f /ansys_inc/fluent.tgz
-
-FROM ${baseImage} as ceibuilder
-COPY cei.tgz /ansys_inc/
-RUN tar -xzf /ansys_inc/cei.tgz --directory /ansys_inc/ && rm -f /ansys_inc/cei.tgz
-
-FROM ${baseImage}
-LABEL description="Ansys Fluent 2022 R2"
-MAINTAINER Laurence Janoo "laurence.janoo@ansys.com"
+# Specify base image
+FROM centos:7
```

Comment:
@seanpearsonuk @mkundu1 

We have tested various Linux distributions only `centos:7` is working fine.

Here is the summary.

```
1. centos:7 - works fine

2. ubuntu:22.04 - 999999 Can't open file cleanup-fluent-*.sh

3. rockylinux:9, fedora:40, almalinux:9 all 3 give following error.

- libcrypt.so.1: cannot open shared object file: No such file or directory -

We have tried to resolve this by installing `libxcrypt.x86_64` but it doesn't help.